compose/.env_slurm_singularity

```diff
@@ -0,0 +1,26 @@
+# This file is setting up a composed Galaxy instance with support
+# for submitting SLURM.
+# Docker compose will use parameters specified in an `.env` file
+# next to the docker-compose.yml file.
+# We recommend to symlink this file and play around with different
+# Galaxy deployments.
+# ln -sf .env_slurm .env
+
+
+# default destination is our SLURM cluster with Singularity enabled
+GALAXY_DESTINATIONS_DEFAULT=slurm_cluster_singularity
+GALAXY_RUNNERS_ENABLE_SLURM=True
+
+# With this Galaxy will search for mulled based images (in /export/database/container_images/singularity/mulled/)
+GALAXY_CONFIG_ENABLE_BETA_MULLED_CONTAINERS=True
+
+GALAXY_CONFIG_TOOL_CONFIG_FILE=config/tool_conf.xml.sample,config/shed_tool_conf.xml,/export/cat_tool_conf.xml
+
+GALAXY_CONFIG_CLEANUP_JOB=onsuccess
+GALAXY_PORT=8080
+
+
+GALAXY_SINGULARITY_VOLUMES=$tool_directory:rw,$working_directory:rw,$default_file_path:rw
```

Comment:
Upps, yes this should be removed. I was too excited :)
It was also the wrong branch, fixed here: https://github.com/bgruening/docker-galaxy-stable/pull/350

Because of the "complicated" I'm playing around with the idea of having a separate .env file just for testing, to keep the user-facing env file clean. What do you think?